Quotient/room.h

```diff
@@ -953,10 +971,21 @@ public Q_SLOTS:
     void pinnedEventsChanged();
     void topicChanged();
     void avatarChanged();
+
+    void memberAdded(QString mxId);
+    void memberNameUpdated(QString mxId);
+    void memberAvatarUpdated(QString mxId);
+    void memberRemoved(QString mxId);
+
+    [[deprecated("Use memberAdded() instead.")]]
     void userAdded(Quotient::User* user);
+    [[deprecated("Use memberRemoved() instead.")]]
     void userRemoved(Quotient::User* user);
+    [[deprecated("Use memberNameUpdated() instead.")]]
```

Comment:
It's currently used in the `UserListModel` of both NeoChat and Quaternion and in both cases the user is removed on `memberAboutToRename` and then added back on `memberRenamed`.  These are the only uses I can find and as far as I see it since all clients would need to patch to use the new infrastructure it's an easy change to just tell the model to refresh the displayName role on the single signal.